<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceInfoDialog</class>
 <widget class="QDialog" name="SourceInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter package source:</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>URI</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Distribution</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Components</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="f_typeCB">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify the type of the source. Supported are &lt;span style=&quot; font-style:italic;&quot;&gt;wpkg&lt;/span&gt; and &lt;span style=&quot; font-style:italic;&quot;&gt;wpkg-src&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>wpkg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>wpkg-src</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="f_uriEdit">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the URI of the source. Can be a &lt;span style=&quot; font-style:italic;&quot;&gt;file:&lt;/span&gt; or &lt;span style=&quot; font-style:italic;&quot;&gt;http:&lt;/span&gt; scheme resource.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Example: file:///usr/local/my_source&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="f_uriButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="f_distEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the distribution of the source.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Example: m2osw&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="f_sourceEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;One or more components, space delimited.&lt;br/&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Example: &amp;quot;main contrib restricted&amp;quot; (without the quotes [&amp;quot;]).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="f_buttonBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;OK&lt;/span&gt;&amp;quot; to accept the changes, &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;Cancel&lt;/span&gt;&amp;quot; to abandon them.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>f_typeCB</tabstop>
  <tabstop>f_uriEdit</tabstop>
  <tabstop>f_uriButton</tabstop>
  <tabstop>f_distEdit</tabstop>
  <tabstop>f_sourceEdit</tabstop>
  <tabstop>f_buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
